app/models/controllr.rb

```diff
@@ -36,11 +36,11 @@ def performance(month, year)
       total: data_totals['monthly_total_hours'].to_f
```

Comment:
It doesn't look like these values are used anywhere (except `total` to check when to stop in `ControllrFetcher`) but I guess it doesn't hurt to leave them in?

`neutral` would need to reconsidered though since we now have a flag for this on projects.

Reply:
If I recall correctly this was introduced for https://github.com/panter/ctrl-dash/.

Should we update `neutral` then to reflect `performance_neutral`? Or add `performance_neutral` as well?

Reply:
Yes that probably makes sense :) I updated Controllr to return the totals for performance-relevant/-neutral hours in the API as well, and return both `holidays` and `neutral` here. But as far as I can tell it's not getting stored anywhere, and `ctrl-dash` doesn't mention `neutral` yet either.